repo/meta/index.json

```diff
@@ -31,6 +31,18 @@
       "versions":{
         "alpha":"0"
       }
+    },
+    {
+      "currentVersion":"1.3.0",
+      "description":"Spark is a fast and general cluster computing system for Big Data",
+      "name":"spark",
+      "tags":[
+        "mesosphere",
+        "framework"
```

Comment:
Nothing yet.  If you have ideas, let's add it to the (currently empty) best-practices.md.